packages/pilot/src/containers/Onboarding/CardOptions/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import {
+  anyPass,
+  isEmpty,
+  is,
+  isNil,
+} from 'ramda'
+import Card from '../Card'
+import styles from './styles.css'
+
+const isFunction = is(Function)
+const isNilOrEmpty = anyPass([isNil, isEmpty])
+
+const CardOptions = ({
+  handleSubmit,
+  images,
+  options,
+}) => (
+  <div className={styles.cardOptionsContainer}>
+    {options.map((option, index) => {
```

Comment:
A variável `options` só é utilizada para pegar _props_ as quais poderiam ser criadas com _parameter destructuring_ ex:
```suggestion
    {options.map(({ description, label, value  }, index) => {
```